src/app/functions/server/l10n.ts

```diff
@@ -78,11 +79,15 @@ export function getL10nBundles(): LocaleData[] {
   }
 
   const languages = acceptLangHeader ? acceptedLanguages(acceptLangHeader) : [];
-  const currentLocales = negotiateLanguages(
-    languages,
-    Object.keys(bundleSources),
-    { defaultLocale: "en" },
-  );
+  const supportedLocales = process.env.SUPPORTED_LOCALES?.split(",");
+  const availableLocales = Object.keys(bundleSources);
+  const filteredLocales =
+    process.env.APP_ENV === "heroku"
+      ? availableLocales
+      : availableLocales.filter((locale) => supportedLocales?.includes(locale));
+  const currentLocales = negotiateLanguages(languages, filteredLocales, {
+    defaultLocale: "en",
+  });
```

Comment:
Not sure if it'd be possible to add `filteredLocales` or `currentLocales` to the Dockerflow <https://monitor.mozilla.org/__version__> endpoint as well so we could confirm that the expected locales are supported in the different environments; although that's probably lower on the priority than a bunch of other backlog bugs.

Reply:
Yeah that would be nice, but we'll first have to figure out how to make that endpoint work in the first place - it's currently always showing the same static value :cry: